crates/rattler/src/install/clobber_registry.rs

```diff
@@ -65,13 +65,19 @@ impl ClobberRegistry {
                 .package_names
                 .iter()
                 .position(|n| n == originating_package)
-                .unwrap();
+                .expect("Package not found even though it was just added");
```

Comment:
It's not that easy (expect doesn't take a format string) and it would be a bit of a performance penalty. This case should _really_ never happen because we add the package names a little further up in the code before adding them to the temp_clobbers.